### PR TITLE
add doc for mutual-ssl authentication validation per API level

### PR DESCRIPTION
**Pull request by tharmini**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/how-tos/security/api-authentication/mutual-ssl-authentication.md [hunk 1]
```diff
@@ -17,6 +17,41 @@ x-wso2-mutual-ssl: "mandatory" # can be "mandatory" or "optional"
 |mandatory| Enable mutual SSL. The client should be verified and the mutual SSL handshake must be passed. the API request will be unauthorized if the handshake is failed.|
 |optional|Disable mutual SSL. Even if the mutual SSL handshake failed, the API request is not necessarily unauthenticated. If the mutual SSL handshake is passed, the request will be filtered by the mutual SSL handler.|
```
Comment (praminda) [inline en/docs/how-tos/security/api-authentication/mutual-ssl-authentication.md:18]:
```suggestion
|optional| Optional mutual SSL. Even if the mutual SSL handshake failed, the API request is not necessarily unauthenticated. If the mutual SSL handshake is passed, the request will be filtered by the mutual SSL handler.|
```

File: en/docs/how-tos/security/api-authentication/mutual-ssl-authentication.md [hunk 2]
```diff
@@ -17,6 +17,41 @@ x-wso2-mutual-ssl: "mandatory" # can be "mandatory" or "optional"
...
 
+
+### Support Mutual SSL Authentication for an API
+
+The Microgateway supports mutual SSL at the API level. It validates the certificate against per API by the name, version, and alias. There are two scenarios in Mutual SSL, the first one is Client directly connects with Microgateway and the second one is the load balancer in front of the Microgateway.
+![Mutual SSL overview]({{base_path}}/assets/img/mutualssl.png)
```
Comment (praminda) [inline en/docs/how-tos/security/api-authentication/mutual-ssl-authentication.md:24]: https://docs.wso2.com/display/MG320/Mutual+SSL+Authentication?preview=/148931521/148931863/mutualssl.svg%20(1).png need to add this to the PR

File: en/docs/how-tos/security/api-authentication/mutual-ssl-authentication.md [hunk 3]
```diff
@@ -17,6 +17,41 @@ x-wso2-mutual-ssl: "mandatory" # can be "mandatory" or "optional"
...
+
+1. <b> Mutual SSL configuration without Load Balancer.</b>
+   
+    Update the mutual SSL configuration in the micro-gw.conf file residing in the <MICROGW_HOME>/conf directory.  Here name is Swagger Petstore, version is 1.0.5, and aliasList is ballerina and wso2apim310.
```
Comment (praminda) [inline en/docs/how-tos/security/api-authentication/mutual-ssl-authentication.md:28]:
```suggestion
    Update the mutual SSL configuration in the micro-gw.conf file residing in the `<MICROGW_HOME>/conf` directory.  Here name is Swagger Petstore, version is 1.0.5, and aliasList is ballerina and wso2apim310.
```

**Review by Rajith90 — APPROVED** [review]

**Comment by Rajith90** [comment]
We will address the review suggestions in a separate PR

**Pull request merged**